{
  "uid" : "5c52f204b4277950",
  "name" : "Secure handling of certificate templates",
  "fullName" : "Certificate Template Security: Secure handling of certificate templates",
  "historyId" : "b3027acd7e47e02b1f41d6bde68b46a0",
  "time" : {
    "start" : 1746705375393,
    "stop" : 1746705375979,
    "duration" : 586
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: STEP: Given a certificate template named \"Sensitive_Template\" exists\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_102_steps.py\", line 27, in step_given_certificate_template_exists\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: STEP: Given a certificate template named \"Sensitive_Template\" exists\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_102_steps.py\", line 27, in step_given_certificate_template_exists\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the admin user is logged into TPC_Web",
      "time" : {
        "start" : 1746705375395,
        "stop" : 1746705375945,
        "duration" : 550
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Given a certificate template named \"Sensitive_Template\" exists",
      "time" : {
        "start" : 1746705375945,
        "stop" : 1746705375946,
        "duration" : 1
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: STEP: Given a certificate template named \"Sensitive_Template\" exists\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_102_steps.py\", line 27, in step_given_certificate_template_exists\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "When an unauthorized user attempts to access it",
      "time" : {
        "start" : 1746705375979,
        "stop" : 1746705375979,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then access should be denied",
      "time" : {
        "start" : 1746705375979,
        "stop" : 1746705375979,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And the attempt should be logged",
      "time" : {
        "start" : 1746705375979,
        "stop" : 1746705375979,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 5,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Certificate Template Security"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 2,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "2f3666ca41eed8cf",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/16/#testresult/2f3666ca41eed8cf",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: Given a certificate template named \"Sensitive_Template\" exists\n",
        "time" : {
          "start" : 1746616926859,
          "stop" : 1746616927446,
          "duration" : 587
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "5c52f204b4277950.json",
  "parameterValues" : [ ]
}